pytorch_lightning/core/lightning.py

```diff
@@ -451,7 +129,7 @@ def __check_not_nested(value: dict, name: str) -> None:
         return value
 
     @staticmethod
-    def __check_allowed(v: Any, name: str, value: Any) -> None:
+    def _check_allowed(v: Any, name: str, value: Any) -> None:
```

Comment:
This is duplicated in `RootLightningModule`, is that right?

Reply:
Yes, removed. Good catch !